boards/arm/mec1501modular_assy6885/doc/index.rst

```diff
@@ -0,0 +1,255 @@
+
+- MEC1501 Modular Card ASSY 6885 Rev A0 `MEC1501 Modular EC Card - Assy_6885 Rev A0p1`
+
+System Clock
+============
+
+The MEC1501 MCU is configured to use the 48Mhz internal oscillator with the
+on-chip PLL to generate a resulting EC clock rate of 12 MHz. See Processor clock
+control register in chapter 4 "4.0 POWER, CLOCKS, and RESETS" of the data sheet in
+the references at the end of this document.
+
+Serial Port
+===========
+UART1 is configured for serial logs.
+
+Jumper settings
+***************
+
+Please follow the jumper settings below to properly demo this
+board. Advanced users may deviate from this recommendation.
+
+Jumper setting for MEC1501 Modular Assy 6885 Rev A1p0
+=================================================
+
+Power-related jumpers
+---------------------
+
+If you wish to power from type A/B connector ``P10`` set the jumper ``JP35 1-2``.
+
+If you wish to power through MECC connector ``P1`` and mate to external platform,
+set the jumper to ``JP35 3-4``.
+
+.. note:: A single jumper is required in JP35.
+
++------+------+------+------+------+------+------+
+| JP30 | JP31 | JP32 | JP33 | JP34 | JP40 | JP21 |
++======+======+======+======+======+======+======+
+| 1-2  | 1-2  | 1-2  | 1-2  |  1-2 | 1-2  | 1-2  |
++------+------+------+------+------+------+------+
+
++------+------+------+------+------+
+| JP6  | JP21 | JP36 | JP27 | JP4  |
++======+======+======+======+======+
+| 2-3  | 1-2  | 1-2  | 2-3  | open |
++------+------+------+------+------+
+
+These jumpers configure nRESETI and JTAG_STRAP respectively.
+
++-----------+--------------+
+| JP22      | JP29         |
+| (nRESETI) | (JTAG_STRAP) |
++===========+==============+
+| 11-12     | 1-2          |
++-----------+--------------+
+
+Boot-ROM Straps.
+----------------
+
+These jumpers configure MEC1501 Boot-ROM straps.
+
++-------------+------------+--------------+-------------+
+| JP37        | J6         | JP41         | JP23        |
+| (CMP_STRAP) | (CR_STRAP) | (VTR2_STRAP) | (BSS_STRAP) |
++=============+============+==============+=============+
+| 1-2         | 1-2        | 1-2          | 3-4         |
++-------------+------------+--------------+-------------+
+
+
+``JP23 3-4`` pulls SHD SPI CS0# up to VTR2. MEC1501 Boot-ROM samples
+SHD SPI CS0# and if high, it loads code from SHD SPI.
+This is the recomended setup.
+
++-------------+------------+----------------------------+
+|  CR_STRAP   | BSS_STRAP  |         SOURCE             |
++-------------+------------+----------------------------+
+|      0      |     X      |   Use 3.3V Private SPI     |
++-------------+------------+----------------------------+
+|      1      |     0      |  Use eSPI Flash channel    |
+|             |     1      |  Use 3.3V Shared channel(R)|
++-------------+------------+----------------------------+
+
+
+Jumper location map
+
++--------------------------------------------------------------------------------------+
```

Comment:
The doc processing sees this drawing as a table  (since it starts with a ``+--------+``, and is not happy about its column and row formatting.  Instead, format this as a code block by taking this whole ASCII drawing and indent it by a space and add a 
``., code-block:: none`` before it.

Reply:
Corrected this and rest of comments, ascii drawing is showing in the html.